src/main/java/net/sf/rails/ui/swing/GameUIManager.java

```diff
@@ -182,6 +182,15 @@ public void init(RailsRoot root, boolean wasLoaded, SplashWindow splashWindow) {
         for (Player player : getPlayers()) {
             currentGuiPlayerNames.add(player.getId());
         }
+
+        localPlayerName = System.getProperty("local.player.name");
+        if (!Util.hasValue(localPlayerName)) {
+            localPlayerName = Config.get("local.player.name");
+        }
+        if (autoSaveLoadStatus > 0) {
+            myTurn = getCurrentPlayer().getId().equals(localPlayerName);
```

Comment:
I know this is kind of independent to this PR:
Isn't `myTurn` only useful in a game that is played with `discord` etc.?
If you do a "normal" game where all players "share" the same screen the variable should always be `true` right?

Reply:
`myTurn` was used (and still is) in the originally polling logic in game manager(s) to determine if/when it should run certain UI updates and calculations.

And yes, if a game is not using the poller then myTurn is always true. This is the reason for this specific check-- we only change `myTurn` if the poller is enabled.

Reply:
And my assumption on the mention of Discord here is not a reference to our Discord and Slack notification classes (which are independent of polling or not) and instead to the user using Discord (or any IM communication tool) to coordinate game play?